catalog/datasets/aat.jsonld

```diff
@@ -25,10 +25,23 @@
       "@type": "DataDownload",
       "contentUrl": "http://vocab.getty.edu/sparql",
       "encodingFormat": "application/sparql-query",
-      "potentialAction": {
-        "@type": "SearchAction",
-        "query": "file://catalog/queries/aat.rq"
-      }
+      "potentialAction": [
+        {
+          "@type": "SearchAction",
+          "query": "file://catalog/queries/search/aat.rq"
+        },
+        {
+          "@type": "FindAction",
+          "query": "file://catalog/queries/lookup/aat.rq",
+          "identifier": [
```

Comment:
To discuss: how can we refer to the source's URI prefix? I have now used `identifier` as property, but this is rather shaky, semantically. Also, I've added support for one or more URI prefixes, just in case a source uses several of them.

Reply:
1. I suggest `schema:url`.

2. Yes.

3. I suggest `<http://vocab.getty.edu/aat> a schema:Dataset ; schema:url <http://vocab.getty.edu/aat/>`, where we interpret `schema:url` as the dataset’s URL _prefix_. In case of multiple prefixes, we can add more than one `schema:url` triple.

Reply:
1. Perfect!
2. Perfect!
3. Perfect!